index.html

```diff
@@ -9,9 +9,11 @@
     <nav>
       <h4>IdeaBox</h4>
       <div class="filter-box">
-        <p class="filter-text">Filter Starred Ideas</p>
-        <button class="nav-button" id="starred-ideas">Show Starred Ideas</button>
-        <button class="nav-button hidden" id="show-all-btn">Show All Ideas</button>
+        <div class="filter-text-style">
```

Comment:
I like this as a way to control the text styling for each of those elements--you could also refactor it by adding your "filter-text-style" class to the div that has the "filter-box" class (on line 11) rather than creating a whole new div